base/commands/viridian/custom_class_list.go

```diff
@@ -0,0 +1,89 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/output"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/viridian"
+)
+
+type CustomClassListCmd struct{}
+
+func (cmd CustomClassListCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("list-custom-classes [cluster-name/cluster-id]")
+	long := `Lists all Custom Classes in the specified Viridian Cluster.
+
+Make sure you login before running this command.
+`
+	short := "List Custom Classes in the specified Viridian Cluster"
+	cc.SetCommandHelp(long, short)
+	cc.SetPositionalArgCount(1, 1)
+	cc.AddStringFlag(propAPIKey, "", "", false, "Viridian API Key")
+
+	return nil
+}
+
+func (cmd CustomClassListCmd) Exec(ctx context.Context, ec plug.ExecContext) error {
+	api, err := getAPI(ec)
+	if err != nil {
+		return err
+	}
+
+	cn := ec.Args()[0]
+
+	csi, stop, err := ec.ExecuteBlocking(ctx, func(ctx context.Context, sp clc.Spinner) (any, error) {
+		sp.SetText("Retrieving custom classes")
+		cs, err := api.ListCustomClasses(ctx, cn)
+		if err != nil {
+			return nil, err
+		}
+		return cs, nil
+	})
+	if err != nil {
+		ec.Logger().Error(err)
+		return fmt.Errorf("error getting custom classes. Did you login?: %w", err)
+	}
+	stop()
+
+	cs := csi.([]viridian.CustomClass)
+	rows := make([]output.Row, len(cs))
+
+	for i, c := range cs {
+		rows[i] = output.Row{
+			output.Column{
+				Name:  "ID",
+				Type:  serialization.TypeInt64,
+				Value: c.Id,
+			},
+			output.Column{
+				Name:  "Name",
+				Type:  serialization.TypeString,
+				Value: c.Name,
+			},
+			output.Column{
+				Name:  "GeneratedFileName",
+				Type:  serialization.TypeString,
+				Value: c.GeneratedFilename,
+			},
+			output.Column{
+				Name:  "Status",
+				Type:  serialization.TypeString,
+				Value: c.Status,
+			},
+			output.Column{
```

Comment:
How is the information in this column is used? Could you only display it only if `--verbose` is true?